packages/perseus/src/widgets/interactive-graphs/interactive-graph-regression.stories.tsx

```diff
@@ -6,261 +6,245 @@ import {mockStrings} from "../../strings";
 import {interactiveGraphQuestionBuilder} from "./interactive-graph-question-builder";
 
 import type {PerseusRenderer} from "@khanacademy/perseus-core";
+import type {Meta, StoryObj} from "@storybook/react";
 
-type StoryArgs = Record<any, any>;
+type Story = StoryObj<typeof MafsQuestionRenderer>;
 
-export default {
+const meta: Meta<typeof MafsQuestionRenderer> = {
```

Comment:
I switched this entire file to CSF v3 so that we'd have typing on the default export as well as trim down the size of all stories (and have better typing on them too!).